scripts/provisioning/s3_start

```diff
@@ -58,6 +58,8 @@ def main():
     elif args.service == 's3bgconsumer':
       # Add action and remove pass
       pass
+    elif args.service == 'haproxy':
+      os.system("/usr/sbin/haproxy -Ws -f /etc/cortx/s3/haproxy.cfg -p /run/haproxy.pid &")
```

Comment:
Please remove the `&` at the end -- single entry point script must "block" and never return, until haproxy itself exits.